lib/vmfloaty/ssh.rb

```diff
@@ -30,7 +30,12 @@ def self.ssh(verbose, host_os, token, url)
       end
 
       hostname = "#{response[host_os]["hostname"]}.#{response["domain"]}"
-      cmd = "#{ssh_path} #{user}@#{hostname}"
+      ssh_options= ["-oStrictHostKeyChecking=no", "-oUserKnownHostsFile=/dev/null"]
+      if ENV['VMFLOATY_SSH_KEY']
+        ssh_options << "-i #{ENV['VMFLOATY_SSH_KEY']}"
```

Comment:
I wonder if this would be better suited as a command line flag, rather than an environment variable? Then when users do `floaty ssh --help` they can see it as an available option.